backend/server.js

```diff
@@ -8,7 +8,7 @@ const port = process.env.PORT || 3000;
 
 const startUpCallback = () => console.log(`listening on port ${port}`);
 
-db.sync(true).then(() => {
+db.sync(false).then(() => {
```

Comment:
is it okay that we keep this true while our database is still changing?

Reply:
Okay, sure it is fine.

Reply:
👍